dist/dev/css/main-index.css

```diff
@@ -17,52 +17,35 @@ body {
         padding-right: 7px;
         padding-left: 7px;
     }
-
-    .col-lg-1 {
-        width: 76px;
-    }
-
-    .col-lg-2 {
-        width: 166px;
+    .container {
+        width: 1066px;
     }
 
-    .col-lg-3 {
-        width: 256px;
-    }
+}
 
-    .col-lg-4 {
-        width: 346px;
+@media (min-width: 992px) and (max-width: 1159px){
+    .col-md-1, .col-md-2, .col-md-3, .col-md-4, .col-md-5, .col-md-6, .col-md-7,
```

Comment:
You have a hole between 1159px and 1200px.

Reply:
but if i made the max width 1200px, then how would it know which width to use?

Reply:
also this https://stackoverflow.com/questions/13241531/what-are-the-rules-for-css-media-query-overlap

Reply:
First of all that doesn't justify the max-width 1159px. The min-width for lg is 1200, but you set the max-width of md to 1159?

Reply:
oh shit you are right

Reply:
lol; change and i'll merge